### ISSUE TITLE
Loads when other websites still load

### ISSUE THREAD

**Issue by rugk** [smorks/keepasshttp-connector#87]
STR:
1. Use a bad DNS server, which does not respond. 😉 
2. Load some website in one tab.
3. In another ([previously] loaded!) website, click on the icon of this add-on.

What happens:
![whenotherwebsitesareloadingiamtoo](https://user-images.githubusercontent.com/11966684/36259201-b9e7efb0-125d-11e8-84d9-203c8f6d066b.png)

What should happen:
It should work independently of the network state.

**Comment by smorks** [comment]
i agree, it should work regardless of network state. i'll look into it.
